ports/espressif/boards/lolin_s3_mini/sdkconfig

```diff
@@ -0,0 +1,48 @@
+#
+# Component config
+#
+#
+# ESP32S3-Specific
+#
+CONFIG_ESP32S3_SPIRAM_SUPPORT=y
+#
+# SPI RAM config
+#
+# CONFIG_SPIRAM_MODE_QUAD=y
```

Comment:
I should have been more specific, sorry.
It's `CONFIG_SPIRAM_TYPE_AUTO` that is unwanted.
You want `CONFIG_SPIRAM_MODE_QUAD` or `CONFIG_SPIRAM_MODE_OCT` enabled.

You can just try out if oct works if you are too lazy to check the schematic.
If oct doesn't boot, quad.